thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.media
+namespace java com.gu.contentatom.thrift.atom.media
+#@namespace scala com.gu.contentatom.thrift.atom.media
+
+typedef i64 Version
+
+enum AssetType {
+  YOUTUBE,
+  FACEBOOK,
+  PODCAST,
+  DAILYMOTION,
+  MAINSTREAM,
+  SUBTITLES
+}
```

Comment:
If we have a generic `URI` Atom, how would the renderer know how to render it? Were you thinking it would be an "iframe", or the `src` attribute for an HTML5 `video` element? I think either way we cannot make it sufficiently generic that it would be used for multiple different render methods. I think we would have to be more specific in its name.